index.html

```diff
@@ -68,58 +68,75 @@ <h2 class="top-subheader">test case management system</h2>
                 </p>
             </div>
 
-            <div class="rTable secondary">
-                <div class="rTableBody">
-                    <div class="rTableRow">
-                        <div class="rTableHead">Everyday testing</div>
-                        <div class="rTableHead">Test management</div>
-                        <div class="rTableHead">Reporting</div>
-                        <div class="rTableHead">Integration</div>
+            <div class="div-table secondary">
+                <div class="div-table-row">
```

Comment:
I think we don't need to have div elements for rows. Otherwise you are duplicating a table element structure but using different tags. Instead specify only the <div class="div-table-col-cell"> elements and use CSS to specify their width, possibly height, padding and alignment. If you do this correctly all of the 4 cels on each row should stack one next to the other and visually look like a table.